ironfish-cli/src/commands/accounts/transaction.ts

```diff
@@ -47,6 +47,10 @@ export class TransactionCommand extends IronfishCommand {
     this.log(`Status: ${response.content.transaction.status}`)
     this.log(`Miner Fee: ${response.content.transaction.isMinersFee ? `✔` : `x`}`)
     this.log(`Fee: ${CurrencyUtils.renderIron(response.content.transaction.fee, true)}`)
+    if (response.content.transaction.blockHash && response.content.transaction.blockSequence) {
+      this.log(`Block Hash: ${response.content.transaction.blockHash}`)
+      this.log(`Block Sequence: ${response.content.transaction.blockSequence}`)
+    }
```

Comment:
It might be helpful to display these rows even if the values are null. Perhaps with some indication that they're null because the transaction hasn't been included in a block?

Reply:
I think this is unnecessary because we have `response.content.transaction.status` to indicate whether the transaction is `pending` `unconfirmed` `expired` etc.